.github/workflows/coverage.yml

```diff
@@ -0,0 +1,128 @@
+# Copyright (C) 2021 Intel Corporation
+# SPDX-License-Identifier: BSD-3-Clause
+
+# https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions
+
+name: delta coverage checks
+
+permissions:
+  # Grant read permissions to repository in case it is not a forked public
+  # repository, but a private repository that was created manually.
+  contents: read
+
+  # Grant read permissions to private container images.
+  packages: read
+
+on:
+  push:
+    paths:
+      - '**'
+      - '!**.md'
+      - '!**/.clang-format'
+      - '!**/COPYING'
+      - '!**/LICENSE'
+      - '!.github/**'
+      - '.github/workflows/coverage.yml'
+      - '!.gitignore'
+      - '!cmake/manifests/**'
+      - 'cmake/manifests/linux/**'
+      - '!container/**'
+      - '!docs/**'
+      - '!scripts/**'
+
+  pull_request:
+    paths:
+      - '**'
+      - '!**.md'
+      - '!**/.clang-format'
+      - '!**/COPYING'
+      - '!**/LICENSE'
+      - '!.github/**'
+      - '.github/workflows/coverage.yml'
+      - '!.gitignore'
+      - '!cmake/manifests/**'
+      - 'cmake/manifests/linux/**'
+      - '!container/**'
+      - '!docs/**'
+      - '!scripts/**'
+
+jobs:
+  build:
+    runs-on: ubuntu-20.04
+
+    container:
+      image: ghcr.io/intel/fpga-runtime-for-opencl/ubuntu-20.04-dev:main
+
+    continue-on-error: true
+
+    env:
+      WORK_DIR: /__w/fpga-runtime-for-opencl/fpga-runtime-for-opencl
+      TMP_DIR: /home/build/

```

Comment:
Please never mind, using `$HOME` is not a good idea because the GitHub runner overrides with a directory, `/github/home` which is writable only to the root container user and differs from the home directory of the user.
- https://github.com/actions/runner/issues/863
- https://github.com/actions/runner/issues/1146